src/Simulation/Simulation.jl

```diff
@@ -260,60 +269,66 @@ function Grid(sim::Simulation{T, Cylindrical};
         end 
     end
 
-    append!(important_r_points, endpoints(world.intervals[1])...)
-    important_r_points = unique!(sort!(important_r_points))
+    append!(important_r_ticks, endpoints(world.intervals[1])...)
+    important_r_ticks = unique!(sort!(important_r_ticks))
     if add_points_between_important_point
-        important_r_points = sort!(vcat(important_r_points, StatsBase.midpoints(important_r_points)))
+        important_r_ticks = sort!(vcat(important_r_ticks, StatsBase.midpoints(important_r_ticks)))
     end
-    iL = searchsortedfirst(important_r_points, world.intervals[1].left)
-    iR = searchsortedfirst(important_r_points, world.intervals[1].right)
-    important_r_points = unique(map(t -> isapprox(t, 0, atol = 1e-12) ? zero(T) : t, important_r_points[iL:iR]))
-    important_r_points = merge_close_ticks(important_r_points)
-    important_r_points = initialize_axis_ticks(important_r_points; max_ratio = T(max_distance_ratio))
-    important_r_points = fill_up_ticks(important_r_points, max_distance_r)
-
-    append!(important_z_points, endpoints(world.intervals[3])...)
-    important_z_points = unique!(sort!(important_z_points))
+    iL = searchsortedfirst(important_r_ticks, world.intervals[1].left)
+    iR = searchsortedfirst(important_r_ticks, world.intervals[1].right)
+    important_r_ticks = unique(map(t -> isapprox(t, 0, atol = 1e-12) ? zero(T) : t, important_r_ticks[iL:iR]))
+    important_r_ticks = merge_close_ticks(important_r_ticks)
+    imp2order_r_ticks = merge_close_ticks(second_order_imp_ticks[1], min_diff = world_Δs[1] / 20)
+    important_r_ticks = merge_second_order_important_ticks(important_r_ticks, imp2order_r_ticks)   
```

Comment:
Isn't `important_r_ticks` supposed to be unique already by this implementation?
(Same thing later for `φ`, `z`, `x` and `y`)?
Do we need the `unique!` statement in `merge_second_order_important_ticks` then?